src/renate/utils/hf_utils.py

```diff
@@ -0,0 +1,44 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+from dataclasses import dataclass
+from typing import Any, Dict, List
+
+from transformers import BatchEncoding, DataCollatorWithPadding
+
+
+@dataclass
+class DataCollatorWithPaddingForWildTime(DataCollatorWithPadding):
+    # This adds to the `transformer` library's DataCollatorWithPadding. That data collator expects
```

Comment:
make this a docstring

Reply:
It doesn't adhere to the traditional docstring convention of a summary + detail separation.

Reply:
Should be possible to reformat it that way, right?